Userland/Libraries/LibWeb/Page/EventHandler.h

```diff
@@ -59,7 +59,7 @@ class EventHandler {
 
     bool m_in_mouse_selection { false };
 
-    WeakPtr<Layout::Node> m_mouse_event_tracking_layout_node;
+    JS::GCPtr<Painting::Paintable> m_mouse_event_tracking_paintable;
```

Comment:
Who visits this?

Reply:
Shouldn't it be safe to assume that paintable tree will exist as long as this pointer does?

Reply:
Why is that a safe assumption?

Reply:
riight, it's better to alwys visit instead of assuming when it comes to GC 😅 added missing visit.